src/components/Header/Header.css

```diff
@@ -22,6 +22,6 @@ header {
 a.logo_title {
   margin-bottom: 40px;
   margin-left: 10px;
-  font-size: 25px;
+  font-size: 48px;
   text-decoration: none;
 }
```

Comment:
Add a newline here.

Reply:
where and why?

Reply:
At the end of this file. The good answer is this -
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file